lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -382,12 +385,42 @@ export class EvaluationClientWrapper {
     verifiableCredentials: VerifiableCredential[]
   ) {
     if (selectResults) {
-      selectResults.verifiableCredentials?.forEach((selectableCredential: VerifiableCredential) => {
+      selectResults.selectableVerifiableCredentials?.forEach((selectableCredential: VerifiableCredential) => {
         const foundIndex: number = verifiableCredentials.findIndex(
           (verifiableCredential) => selectableCredential.id === verifiableCredential.id
         );
         selectResults.vcIndexes?.push(foundIndex);
       });
     }
   }
+
+  public determineAreRequiredCredentialsPresent(matchSubmissionRequirements: SubmissionRequirementMatch[]): Status {
+    let status = Status.INFO;
+    if (!matchSubmissionRequirements || !matchSubmissionRequirements.length) {
+      status = Status.ERROR;
+    }
+    for (const m of matchSubmissionRequirements) {
+      if (m.matches.length == 0 && (!m.from_nested || m.from_nested.length == 0)) {
+        return Status.ERROR;
+      }
+      if (m.count && m.matches.length < m.count && (!m.from_nested || !m.from_nested?.length)) {
+        return Status.ERROR;
+      }
+      if (m.count && (m.matches.length > m.count || (m.from_nested && m.from_nested?.length > m.count))) {
+        status = Status.WARN;
```

Comment:
This will be an error. Not a warning.

Reply:
why? it's like the `max` thing, both should be `warning`

Reply:
Well in case of the const. the DIF Spec is more strict. It says it is an error. But I am fine with calling it a warning.